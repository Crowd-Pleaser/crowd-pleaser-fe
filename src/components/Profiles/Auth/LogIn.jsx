import React, { useState } from 'react';
import { postLogin } from '../../../services/auth-api';

export default function LogIn() {
  const [email, setEmail] = useState('');
  const [password, setPassword] = useState('');

  const handleLogIn = event => {
    event.preventDefault();
    postLogin(email, password);
  };
  return (
    <div>
      <h2>Log In</h2>
      <form onSubmit={handleLogIn}>
        <input type="text" value={email} onChange={({ target }) => setEmail(target.value)} placeholder="Email address" />
        <input type="password" value={password} onChange={({ target }) => setPassword(target.value)} placeholder="Password" />
        <button>Log In</button>
      </form>
    </div>
  );
}
