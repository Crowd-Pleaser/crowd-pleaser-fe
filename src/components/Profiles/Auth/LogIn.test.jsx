import React from 'react';
import { shallow } from 'enzyme';
import LogIn from './LogIn';

describe('LogIn', () => {
  it('matches a LogIn snapshot', () => {
    const wrapper = shallow(<LogIn />);
    expect(wrapper).toMatchSnapshot();
  })
})