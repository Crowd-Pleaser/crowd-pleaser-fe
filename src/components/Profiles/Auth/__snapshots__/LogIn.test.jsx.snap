// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogIn matches a LogIn snapshot 1`] = `
<div>
  <h2>
    Log In
  </h2>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      placeholder="Email address"
      type="text"
      value=""
    />
    <input
      onChange={[Function]}
      placeholder="Password"
      type="password"
      value=""
    />
    <button>
      Log In
    </button>
  </form>
</div>
`;
